### ISSUE TITLE
Gatk4Mutect2 usage

### ISSUE THREAD

**Issue by drtconway** [PMCC-BioinformaticsCore/janis-bioinformatics#40]
Hi Janis,

According to the Mutect2 [documentation](https://gatk.broadinstitute.org/hc/en-us/articles/360042477952-Mutect2), mutect2 can be run with tumor only BAMs:
```
 gatk Mutect2 \
   -R reference.fa \
   -I sample.bam \
   -O single_sample.vcf.gz
```

However, when I attempt to do this, the Janis binding objects:
```
$ janis run -o data/results Gatk4Mutect2 --reference reference/hg19.fa.gz --tumorBams data/PRJ190532_CAS0057_3.bam
2020-05-28T02:08:09 [INFO]: Starting task with id = 'e5b47c'
e5b47c
2020-05-28T02:08:09 [INFO]: Exporting tool files to '/data/results/janis/workflow/original'
2020-05-28T02:08:09 [INFO]: Zipping tools
2020-05-28T02:08:09 [INFO]: Zipped tools
2020-05-28T02:08:09 [INFO]: Saved workflow with id 'Gatk4Mutect2' to '/data/results/janis/workflow'
2020-05-28T02:08:09 [WARN]: Could not find a cromwell process to end, SKIPPING
Traceback (most recent call last):
  File "/usr/local/bin/janis", line 11, in <module>
    load_entry_point('janis-pipelines.runner==0.9.17', 'console_scripts', 'janis')()
  File "/usr/local/lib/python3.8/dist-packages/janis_pipelines.runner-0.9.17-py3.8.egg/janis_assistant/cli.py", line 153, in process_args
  File "/usr/local/lib/python3.8/dist-packages/janis_pipelines.runner-0.9.17-py3.8.egg/janis_assistant/cli.py", line 792, in do_run
  File "/usr/local/lib/python3.8/dist-packages/janis_pipelines.runner-0.9.17-py3.8.egg/janis_assistant/main.py", line 491, in fromjanis
  File "/usr/local/lib/python3.8/dist-packages/janis_pipelines.runner-0.9.17-py3.8.egg/janis_assistant/main.py", line 461, in fromjanis
  File "/usr/local/lib/python3.8/dist-packages/janis_pipelines.runner-0.9.17-py3.8.egg/janis_assistant/management/configmanager.py", line 189, in start_task
  File "/usr/local/lib/python3.8/dist-packages/janis_pipelines.runner-0.9.17-py3.8.egg/janis_assistant/management/workflowmanager.py", line 189, in from_janis
  File "/usr/local/lib/python3.8/dist-packages/janis_pipelines.runner-0.9.17-py3.8.egg/janis_assistant/management/workflowmanager.py", line 658, in prepare_and_output_workflow_to_evaluate_if_required
  File "/usr/local/lib/python3.8/dist-packages/janis_pipelines.runner-0.9.17-py3.8.egg/janis_assistant/modifiers/base.py", line 34, in apply_many
  File "/usr/local/lib/python3.8/dist-packages/janis_pipelines.runner-0.9.17-py3.8.egg/janis_assistant/modifiers/base.py", line 20, in apply
  File "/usr/local/lib/python3.8/dist-packages/janis_pipelines.runner-0.9.17-py3.8.egg/janis_assistant/modifiers/inputchecker.py", line 30, in inputs_modifier
  File "/usr/local/lib/python3.8/dist-packages/janis_pipelines.runner-0.9.17-py3.8.egg/janis_assistant/utils/__init__.py", line 216, in validate_inputs
ValueError: There were errors in 2 inputs: {'normalBams': 'value was null', 'normalSample': 'value was null'}
$ 
```

**Comment by illusional** [comment]
Hi @drtconway, thanks for the report. I've addressed your problems in the mentioned PR #41. I'm happy to release janis-bioinformatics if you need it to be released.

**Comment by drtconway** [comment]
Thanks for the quick response! It's not urgent at the moment. I was just trying things out.
